### PR TITLE
Test case for wt3 solution-pattern updates

### PR DESCRIPTION
**Pull request by tonyxrmdavidson**
# Description
Test-case for wt3 solution-pattern update

### CODE REVIEW

**Comment by pawelpaszki** [comment]
It looks good, but `npm run lint` fails. Can you please fix it?

File: test-cases/package.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
     "form-data": "3.0.0",
     "googleapis": "47.0.0",
     "gray-matter": "4.0.2",
-    "handlebars": "4.1.2",
+    "handlebars": "^4.7.6",
```
Comment (damienomurchu) [inline test-cases/package.json:20]: @tonyxrmdavidson is this required?

**Comment by pawelpaszki** [comment]
we need to merge one more change before this can be merged. I will approve it after that
/hold

**Pull request merged**